nixos/modules/security/acme/default.nix

```diff
@@ -974,12 +969,6 @@ in {
 
       users.groups.acme = {};
 
-      # for lock files, still use tmpfiles as they should better reside in /run
-      systemd.tmpfiles.rules = [
-        "d ${lockdir} 0700 ${user} - - -"
-        "Z ${lockdir} 0700 ${user} - - -"
-      ];
```

Comment:
I'm not sure about this anymore, using the setup service's runtime dir feels like spooky action at a distance and less declarative since the dir is not actually used by that service.

Reply:
I don't see how it's spooky? We have no explicit dependency between tmpfiles setup and the creation of the lockdir. By using acme-setup.service's runtime directory, systemd will guarantee it exists and is writable when the service starts + subsequently when the renewals happen. It also guarantees automatic cleanup.

Generally, it is expected that /run/ directories map to RuntimeDirectory entries in systemd services.

Reply:
Spooky because the relation is unclear between where the dir is created and where it's used, especially without the variable reference.

As [discussed on matrix](https://matrix.to/#/!MthpOIxqJhTgrMNxDS:nixos.org/$7o9seC_S3S8etPo_mkx3tRxZfr3g8fnWHjHNrEeSdlI), tmpfiles is actually not ordered before the other services so this prevents a race.
Fine by me as is :)